infra/gcp/main.tf

```diff
@@ -237,37 +307,70 @@ resource "google_sql_database_instance" "db" {
       private_network = google_compute_network.default.id
       require_ssl = true
     }
+
+    backup_configuration {
+      binary_log_enabled             = false
+      enabled                        = true
+      location                       = "us"
+      point_in_time_recovery_enabled = false
+      start_time                     = "13:00"
+      transaction_log_retention_days = 7
+
+      backup_retention_settings {
+	retained_backups = 7
+	retention_unit   = "COUNT"
+      }
+    }
+
     database_flags {
-      name = "innodb_log_buffer_size"
+      name  = "innodb_log_buffer_size"
       value = "536870912"
     }
     database_flags {
-      name = "innodb_log_file_size"
+      name  = "innodb_log_file_size"
       value = "5368709120"
     }
     database_flags {
-      name = "event_scheduler"
+      name  = "event_scheduler"
       value = "on"
     }
     database_flags {
-      name = "skip_show_database"
+      name  = "skip_show_database"
       value = "on"
     }
     database_flags {
-      name = "local_infile"
+      name  = "local_infile"
       value = "off"
     }
+
+    insights_config {
+      query_insights_enabled  = true
+      query_string_length     = 1024
+      record_application_tags = false
+      record_client_address   = false
+    }
+
+    location_preference {
+      zone = "us-central1-a"
+    }
+
+    maintenance_window {
+      day  = 7
+      hour = 16
+    }
   }
+
+  timeouts {}
 }
 
 resource "google_compute_address" "gateway" {
-  name = "gateway"
+  name = "site"
```

Comment:
Why this name change?

Reply:
That's the actual name of this address. see `gcloud compute addresses list`

Reply:
I think I'm confused as to why this change is in `infra/gcp/main.tf` instead of us having a different resource in `broad-gcp` and then trying to migrate that one to match what's here.